x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/integrations/integrations_check_complete_helpers.ts

```diff
@@ -44,16 +48,14 @@ export const getIntegrationList = async (
       return { items: emptyItems };
     });
 
-  const installedPackages = packageData?.items?.filter((pkg) => {
-    const integrationCardId = `epr:${pkg.id}`;
+  const installedPackages = installedPackageData?.items?.filter((installedPkg) => {
```

Comment:
<img width="2556" alt="Screenshot 2025-04-25 at 16 09 36" src="https://github.com/user-attachments/assets/ff2f96a0-642e-4099-abd6-0a48d8ed7153" />
<img width="2559" alt="Screenshot 2025-04-25 at 15 57 23" src="https://github.com/user-attachments/assets/a5a61c3a-7c62-4e46-a881-9df3fd45111c" />
Hello @kpollich @nchaulet I have a question for the returned result of the `installed integration api` with `showOnlyActiveDatastream=true`. When I ran it locally, I could see that most of the integrations's `dataStream` is an empty array. Before I installed `ElasticDefend` locally, it appears as `installed` already. However, if we filter the installed integrations by non empty dataStream, the integration (e.g. SentinelOne) I just installed would be excluded.

Reply:
Hey I am not sure to totally understand the question but `showOnlyActiveDataStreams` will return in datastreams only the datastream where we actually have data, in opposite to the one provided by the integration. 
So if you just installed an integration like `SentinelOne` and no data has been ingested yet it's expected that datastreams array will be empty

Reply:
Right @nchaulet , are we able to explain the behaviour of `ElasticDefend` in this case?

Reply:
@nchaulet Do you mean I should filter the api response by `dataStream.length > 0` here? and for the integrations like `SentinelOne` it's expected that it will remain uninstalled even I have clicked the `installed` button from the Fleet page but no data has been ingested?

Do you know are all the integrations appear in the first image (the one with dark theme) behave like this? (meaning clicking instal button !==  adding data)?

Reply:
Basically here we only want the integrations appear as `Active` if user clicked the `install button` from the integration page.

Reply:
Hello @paulewing , Please have a look to this. If we mark integrations `Active` only when integrations have data added, The preinstalled integrations will be excluded, however some integrations like `crowdstrike` and `SentinelOne` won't appear status after clicking their `install` buttons. As it hasn't got data added, is this acceptable?


https://github.com/user-attachments/assets/2086abb2-b1bb-4fdd-b6d8-98f33f5749ef

Reply:
> @nchaulet Do you mean I should filter the api response further by dataStream.length > 0 here? and for the integrations like SentinelOne is it expected that it will remain uninstalled even I have clicked the installed button from the Fleet page but no data has been ingested?
Do you know are all the integrations appear in the first image (the one with dark theme) behave like this? (meaning clicking instal button !== adding data)?

Yes it will be the same for all integration until agents report data, there will be not datastreams.